app/feemarketkeeper.go

```diff
@@ -0,0 +1,30 @@
+package app
+
+import (
+	"math/big"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	evmtypes "github.com/tharsis/ethermint/x/evm/types"
+	feemarkettypes "github.com/tharsis/ethermint/x/feemarket/types"
+)
+
+const DefaultGasPrice int64 = 5000000000000
+
+var _ evmtypes.FeeMarketKeeper = &ConstantFeeMarketKeeper{}
+
+// ConstantFeeMarketKeeper implements a fee market keeper that returns a constant base fee
+type ConstantFeeMarketKeeper struct{}
+
+func (fmk ConstantFeeMarketKeeper) GetBaseFee(ctx sdk.Context) *big.Int {
```

Comment:
any reason you are replacing the feemarketkeeper by a custom one instead of using the one provided by ethermint with fix base fee?

Reply:
I thought it could be simpler to upgrade if we don't include the feemarket module, but maybe no need for the extra trouble. Just added the module.

Reply:
changed to module.